types/tizen-common-web/index.d.ts

```diff
@@ -1,84 +1,14699 @@
+     *
+     */
+    readonly capabilities: ReadonlyArray<string>;
+    /**
+     * Logical (translatable) display name.
+     */
+    readonly displayName: string;
+    /**
+     * Path to the icon representing the account provider.
+     */
+    readonly iconUri: string;
+    /**
+     * Boolean value that indicates whether multiple accounts are supported.
+     */
+    readonly isMultipleAccountSupported: boolean;
+    /**
+     * Path to the small icon representing the account provider.
+     */
+    readonly smallIconUri: string;
+}
+/**
+ * The Alarm interface is an abstract interface for alarm types.
+ */
+export class Alarm {
+    constructor();
+    /**
+     * The alarm identifier.
+     */
+    readonly id: AlarmId | null;
+}
+export interface Alarm {
+    prototype: Alarm;
+    new(): Alarm; // tslint:disable-line:no-misused-new
+}
+/**
+ * The AlarmAbsolute interface provides an absolute alarm, which triggers at a specified absolute date.
+ *
+ * If a _period_ is provided, the alarm keeps triggering for the given interval. If the _daysOfTheWeek_ array is not empty, the alarm triggers every week, for the given days, at the time defined by the _date_ attribute.
+ */
+export class AlarmAbsolute extends Alarm {
+    constructor(date: Date); // tslint:disable-line:unified-signatures
+    constructor(date: Date, daysOfTheWeek: ByDayValue[]); // tslint:disable-line:unified-signatures
+    constructor(date: Date, period: number); // tslint:disable-line:unified-signatures
+    /**
+     * An attribute to store the absolute date/time when the alarm is initially triggered.
+     *
+     * This attribute is precise to the second. Milliseconds will be ignored.
+     */
+    readonly date: Date;
+    /**
+     * An attribute to store the days of the week associated with the recurrence rule.
+     *
+     * By default, this attribute is set to an empty array.
+     * The _period_ and _daysOfTheWeek_ attributes are mutually exclusive.
+     */
+    readonly daysOfTheWeek: ReadonlyArray<ByDayValue>;
+    /**
+     * An attribute to store the duration in seconds between each trigger of the alarm.
+     *
+     * By default, this attribute is set to null, indicating that this alarm does not repeat.
+     * The _period_ and _daysOfTheWeek_ attributes are mutually exclusive.
+     * @note _deprecated_ 4.0 Since Tizen 4.0 constructor AlarmAbsolute(Date date, long period) is deprecated, thus this attribute should not be used.
+     */
+    readonly period: number | null;
+    /**
+     * Returns the date / time of the next alarm trigger.
+     *
+     * If the alarm has expired, this method returns null. The returned date is precise to the second.
+     *
+     * @returns The date/time of the next alarm trigger.
+     *
+     * @throw WebAPIException with error type UnknownError, if the method cannot be completed because of an unknown error.
+     */
+    getNextScheduledDate(): Date | null;
+}
+export interface AlarmAbsolute {
```

Comment:
Another case of incorrectly declaring the *static* (i.e., constructor) side of the class on an interface that represents the *instance*. Please remove.